src/clj/puppetlabs/services/jruby/jruby_puppet_agents.clj

```diff
@@ -1,34 +1,21 @@
 (ns puppetlabs.services.jruby.jruby-puppet-agents
-  (:import (clojure.lang IFn Agent)
-           (puppetlabs.services.jruby.jruby_puppet_core PoisonPill RetryPoisonPill)
-           (com.puppetlabs.puppetserver PuppetProfiler))
+  (:import (clojure.lang IFn)
+           (com.puppetlabs.puppetserver PuppetProfiler)
+           (puppetlabs.services.jruby.jruby_puppet_schemas PoisonPill RetryPoisonPill))
```

Comment:
`defrecord` ultimately results in creating a Java class behind the scenes, so this is how you have to reference them as far as I understand.  (Cursive just did this for me though, so I'm slightly less than 100% certain about that.)